src/App.js

```diff
@@ -1,5 +1,90 @@
+import { MissionUtils } from "@woowacourse/mission-utils";
+
 class App {
-  async play() {}
+
+  getRandomNumber(){
+    const computerNumber = [];
+    while (computerNumber.length < 3){
+      const number = MissionUtils.Random.pickNumberInRange(1, 9);
+      if(!computerNumber.includes(number))
+        computerNumber.push(number);
+    }
+    return computerNumber;
+  }
+
+  async getUserInput(){
+    const userInput = await MissionUtils.Console.readLineAsync("숫자를 입력해주세요 : ");
+    const validatedUserInput = this.checkInputValidation(userInput);
+
+    return validatedUserInput;
+  }
+
+  checkInputValidation(userInputNumber){
+    const userInputNumberArr = userInputNumber.split('').map(Number);
```

Comment:
매핑할 때 `Array.from()` 을 사용하면 중간 배열 생성을 방지할 수 있어요!

```suggestion
    const userInputNumberArr = Array.from(userInputNumber, Number);
```

[참고 자료]
https://github.com/parksb/javascript-style-guide#arrays--mapping